concent_api/core/message_handlers.py

```diff
@@ -803,6 +804,16 @@ def handle_send_force_payment(
         *[subtask_results_accepted.task_to_compute for subtask_results_accepted in client_message.subtask_results_accepted_list],
     )
 
+    if not are_all_payment_ts_younger_than_last_payment_closure_time_if_payment_exists(  # pylint: disable=no-value-for-parameter
```

Comment:
Only Bankster should call SCI methods. This has to be moved to Bankster and raise exception catched by handler, then return `ForcePaymentRejected`.

In the issue description there is
> Yes. It's one of the checks before doing the calculations but since it needs information from the blockchain, it must be inside Bankster.

Reply:
done